tests/api_tests/base/test_views.py

```diff
@@ -47,6 +53,9 @@ def test_view_classes_have_minimal_set_of_permissions_classes(self):
         view_modules = [name for _, name, _ in pkgutil.iter_modules(['api'])]
         for module in view_modules:
             for name, view in inspect.getmembers(sys.modules['api.{}.views'.format(module)], inspect.isclass):
+                if getattr(view, 'permissions_exempt', False):
```

Comment:
One alternative might be to modify our test case that does these permissions checks to use the `urls.py` files instead of `views.py`.

That way we just look at the views that are actually wired to to functional routes. 

I'll give this a try.